test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/https/WorkbenchHttpsIntegrationTest.java

```diff
@@ -39,46 +39,29 @@
 import org.apache.http.impl.client.HttpClients;
 import org.apache.http.ssl.SSLContextBuilder;
 import org.assertj.core.api.Assertions;
-import org.junit.After;
-import org.junit.Before;
 import org.junit.Test;
 import org.kie.cloud.api.DeploymentScenarioBuilderFactory;
-import org.kie.cloud.api.DeploymentScenarioBuilderFactoryLoader;
 import org.kie.cloud.api.scenario.WorkbenchWithKieServerScenario;
-import org.kie.cloud.common.logs.InstanceLogUtil;
-import org.kie.cloud.maven.constants.MavenConstants;
+import org.kie.cloud.integrationtests.AbstractCloudIntegrationTest;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class WorkbenchHttpsIntegrationTest {
+public class WorkbenchHttpsIntegrationTest extends AbstractCloudIntegrationTest<WorkbenchWithKieServerScenario> {
 
     private static final String organizationalUnitRestRequest = "rest/organizationalunits";
     private static final String ORGANIZATION_UNIT_NAME = "myOrgUnit";
     private static final String WORKBENCH_LOGIN_SCREEN_TEXT = "Sign In";
 
-    private static final DeploymentScenarioBuilderFactory deploymentScenarioFactory = DeploymentScenarioBuilderFactoryLoader.getInstance();
-    private WorkbenchWithKieServerScenario workbenchWithKieServerScenario;
-
     private static final Logger logger = LoggerFactory.getLogger(WorkbenchHttpsIntegrationTest.class);
 
-    @Before
-    public void setUp() {
-        workbenchWithKieServerScenario = deploymentScenarioFactory.getWorkbenchWithKieServerScenarioBuilder()
-                .withExternalMavenRepo(MavenConstants.getMavenRepoUrl(), MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
-                .build();
-        workbenchWithKieServerScenario.deploy();
-    }
-
-    @After
-    public void tearDown() {
-        InstanceLogUtil.writeDeploymentLogs(workbenchWithKieServerScenario.getWorkbenchDeployment());
-        InstanceLogUtil.writeDeploymentLogs(workbenchWithKieServerScenario.getKieServerDeployment());
-        workbenchWithKieServerScenario.undeploy();
+    @Override
+    protected WorkbenchWithKieServerScenario createDeploymentScenario(DeploymentScenarioBuilderFactory deploymentScenarioFactory) {
```

Comment:
We can provide default implementation in AbstractCloudIntegrationTest. In that case only the tests who needs specific configuration will override the method implementation.
But this will require changes in cloud API. I think we should implement that in separate PR.